p/themes/Mapco/_components.css

```diff
@@ -0,0 +1,289 @@
+/* stylelint-disable property-no-vendor-prefix */
```

Comment:
Where does this file comes from? Maybe an error?

Reply:
sorry. I will delete it.